salt_stack/salt/python/python36.sls

```diff
@@ -0,0 +1,49 @@
+# Build Python 3.6.x from source and install to /usr/local/bin.
```

Comment:
I'm package installing Python 3.6 on the build server (also in `salt_stack/salt/lambda-dev/init.sls`).